crates/sncast/src/response/print.rs

```diff
@@ -163,6 +164,11 @@ fn apply_numbers_formatting(value: OutputValue, formatting: NumbersFormat) -> Ou
     }
 }
 
+pub fn print_as_warning(error: &Error) {
+    let warning_tag = style("Warning:").color256(11);
+    println!("{warning_tag} {error}");
```

Comment:
```suggestion
    let warning_tag = style("Warning").color256(11);
    println!("{warning_tag}: {error}");
```
just a suggestion, the colon wouldn't be colored